src/main/java/core/mate/academy/model/Bulldozer.java

```diff
@@ -4,12 +4,22 @@
  * Add some custom fields that could be only in Bulldozer
  * Do not remove no-args constructor
  */
-public class Bulldozer extends Machine {
+
```

Comment:
Consider adding a JavaDoc comment for the `Bulldozer` class to describe its purpose and any specific details about its fields or behavior. This is important for maintaining good documentation practices.